### PR TITLE
Dev kt ja tk257

### PR DESCRIPTION
**Pull request by karuna835**
agent name and lead owner filter and added in listing and excel.

### CODE REVIEW

File: app/apis/webapi/backend/dash/package.export.php [hunk 1]
```diff
@@ -328,7 +330,20 @@ function where_clause($name, $operator, $value, &$where) {
...
+			if(!empty($lead_owner_data))
+			{
+				$package["lead_owner"] = $lead_owner_data[0]['name'];
+			}
+			else
+			{
+				$package["lead_owner"] = "";
+			}
+		}
+
```
Comment (kailasb) [inline app/apis/webapi/backend/dash/package.export.php:346]: use ternary operator

File: app/apis/webapi/backend/dash/package.query.php [hunk 2]
```diff
@@ -126,8 +126,29 @@ function where_clause($name, $operator, $value, &$where, $joiner="") {
...
+			}
+
+		}
+		if($user_found){
+			$agent_where = "(package.creator_id IN (".implode(",", $users).") OR package.lead_owner_id IN (".implode(",", $users)."))";
+			if(!empty($where)){
+				$where.= ' AND '.$agent_where;
+			}else{
+				$where =$agent_where;
+			}
```
Comment (kailasb) [inline app/apis/webapi/backend/dash/package.query.php:147]: use ternary operator

File: app/templates/jaws/backend/dash/assets/js/controllers/payment.track.js [hunk 3]
```diff
@@ -32,6 +32,9 @@ angular.module('jaws')
             func = func.substr(0, func.indexOf('('));
             return func;
         }
+		$scope.onCategoryChange = function(){
+			
+		}
```
Comment (kailasb) [inline app/templates/jaws/backend/dash/assets/js/controllers/payment.track.js:37]: what the use of this code

**Review by kailasb — CHANGES_REQUESTED** [review]
check feedback

File: app/apis/webapi/backend/dash/package.export.php (hunk 1 above)
Comment (karuna835) [reply app/apis/webapi/backend/dash/package.export.php]: added ternary operator

File: app/apis/webapi/backend/dash/package.query.php (hunk 2 above)
Comment (karuna835) [reply app/apis/webapi/backend/dash/package.query.php]: used ternary operator

File: app/templates/jaws/backend/dash/assets/js/controllers/payment.track.js (hunk 3 above)
Comment (karuna835) [reply app/templates/jaws/backend/dash/assets/js/controllers/payment.track.js]: removed code

**Review by kailasb — APPROVED** [review]
changes are proper

**Pull request merged**